### ISSUE TITLE
Whether there is an inconsistency way to calculated [user] time under Windows and Linux?

### ISSUE THREAD

**Issue by fyy1996** [fraunhoferhhi/vvenc#195]
Hi~
If we run a multi-threaded instance under windows and Linux, the result of vvencFFapp output time are quite different.  Whether there is an inconsistency way to calculated [user] time under Windows and Linux?
Under Linux, [user] time calculates the CPU time and [elapsed] time calculates the real time, which makes the two results far apart. However, the [user] time is close to the [elapsed] time under windows.
For example, when we encoded the same sequence, the output time under windows shows:
  Total Time:     2444.479 sec. [user]     2444.479 sec. [elapsed]
the output time under Linux shows:
Total Time:     8355.226 sec. [user]     2199.891 sec. [elapsed]

**Comment by adamjw24** [comment]
Yes. On Linux the one output reports CPU time and the other the wallclock time. The former will be larger than the latter in multi-threading encoding. On Windows both outputs report the wallclock time.

If someone knows a fix this would be highly appreciated.
